src/uwtools/tests/drivers/test_mpassit.py

```diff
@@ -0,0 +1,95 @@
+"""
+MPASSIT driver tests.
+"""
+
+from copy import deepcopy
+from pathlib import Path
+from unittest.mock import patch
+
+import f90nml  # type: ignore[import-untyped]
+import yaml
+from pytest import fixture, mark
+
+from uwtools.drivers import mpassit
+from uwtools.drivers.mpassit import MPASSIT
+from uwtools.tests.test_schemas import MPASSIT_CONFIG
+
+# Fixtures
+
+
+@fixture
+def config(tmp_path):
+    outfile = "MPAS-A_out.{{ (cycle + leadtime).strftime('%Y-%m-%d_%H:%M:%S') }}.nc"
+    mpassit_config: dict = {"mpassit": deepcopy(MPASSIT_CONFIG)}
+    mpassit_config["mpassit"]["rundir"] = str(tmp_path)
+    mpassit_config["mpassit"]["namelist"]["update_values"]["config"]["output_file"] = outfile
+    return mpassit_config
```

Comment:
Here are the light mods to the MPASSIT_CONFIG I'm borrowing from test_schema.py.